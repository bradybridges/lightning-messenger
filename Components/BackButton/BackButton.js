import React from 'react';
import {
  TouchableOpacity,
  Image,
  StyleSheet,
  Dimensions,
  StatusBar,
} from 'react-native';

const backIcon = require('../../assets/back-arrow-white.png');

const BackButton = ({ close }) => (
  <TouchableOpacity style={styles.container} onPress={close}>
    <Image style={styles.image} source={backIcon} />
  </TouchableOpacity>
);

const styles = StyleSheet.create({
  container: {
    width: '10%',
    height: '100%',
    display: 'flex',
    justifyContent: 'center',
    alignItems: 'center',
  },
  image: {
    width: 30,
    height: 30,
  },
});

export default BackButton;
