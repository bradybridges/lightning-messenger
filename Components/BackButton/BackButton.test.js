import React from 'react';
import { shallow } from 'enzyme';
import BackButton from './BackButton';

describe('BackButton', () => {
  let wrapper;
  it('should m atch the snapshot', () => {
    wrapper = shallow(<BackButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
