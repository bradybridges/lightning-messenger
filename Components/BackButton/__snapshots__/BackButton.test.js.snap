// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton should m atch the snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100%",
      "justifyContent": "center",
      "width": "10%",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 30,
        "width": 30,
      }
    }
  />
</TouchableOpacity>
`;
